target/portfolio-1/js/playGame.js

```diff
@@ -0,0 +1,146 @@
+async function initMapPlayGame() {
```

Comment:
Why do we have two files "initMapPlayGame"?

src/main/webapp/js/playGame.js 
target/portfolio-1/js/playGame.js

Reply:
src/main/webapp/js/playGame.js is the actual fine and I think target/portfolio-1/js/playGame.js is just a file that is generated everytime mvn package appengine:run.

All files that are in target/ can be ignored.